.github/workflows/ci.yml

```diff
@@ -19,168 +19,12 @@ concurrency:
 
 
 jobs:
-
+  # TODO: replace to @master after the PR merging
```

Comment:
No, not master, just use the local in-repo path.

Reply:
Does it work? 
https://docs.github.com/en/actions/learn-github-actions/reusing-workflows#calling-a-reusable-workflow tells about absolute paths only.
If local paths work -- I'm happy to switch.

Sorry, should go now, return to the PR in the evening.

Reply:
I haven't experimented with that part yet. But since it's in beta, I'd assume that some of the features are undocumented and need discovery.

Reply:
OTOH I've been meaning to have these in a separate repository for reuse across projects.

Reply:
workflow version (`@version`) is mandatory now.